beacon-chain/core/altair/sync_committee_test.go

```diff
@@ -337,10 +337,10 @@ func Test_ValidateSyncMessageTime(t *testing.T) {
 			wantedErr: "",
 		},
 		{
-			name: "sync_message.slot == current_slot+CLOCK_DISPARITY+200ms",
+			name: "sync_message.slot == current_slot+CLOCK_DISPARITY+500ms",
```

Comment:
Where does 500ms come from? The genesisTime arg seems to apply 1000ms

Reply:
It is very flaky from 500 ms - 1000ms onwards. For 500 ms, I ran it 100 times and it would fail 80% of the time, the only number at which it passed without failures was 1000ms.

Reply:
Could you amend this name to match the arguments? It seems like it was `+200ms` and now it's `-1000ms`?

Reply:
```suggestion
			name: "sync_message.slot == current_slot+CLOCK_DISPARITY-1000ms",
```